core/modules/filters/matches.js

```diff
@@ -0,0 +1,46 @@
+/*\
+title: $:/core/modules/filters/matches.js
+type: application/javascript
+module-type: filteroperator
+
+Filter operator for comparing the contents of an input tiddlers field and the
```

Comment:
The difference to **contains** and **intersects** escapes me. Needs to be demoed and documented. And if there is none, it should simply be aliases.

Reply:
contains means the input list contains everything on the operand list, intersects means that the input list and operand list share at least one element.